### PR TITLE
Add documentation for SSM

### LINKED ISSUE #496: Add documentation for SSM client

### PR DESCRIPTION
**Pull request by jderusse**
fixes #496

### CODE REVIEW

**Comment by Nyholm** [comment]
Could you also update the Readme of the SSM client?

File: docs/clients/ssm.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+### Store an encrypted parameter
+
+```php
+use AsyncAws\Ssm\Enum\ParameterType;
+use AsyncAws\Ssm\Input\PutParameterRequest;
+use AsyncAws\Ssm\SsmClient;
+
+$ssm = new SsmClient();
+$parameters = $ssm->putParameter(new PutParameterRequest([
+    'Name' => '/projects/web-site/database_password',
```
Comment (Nyholm) [inline docs/clients/ssm.md:41]:
```suggestion
    'Name' => '/projects/com.website/database_password',
```

Comment (jderusse) [reply docs/clients/ssm.md]: renamed to `/website/`

Comment (Nyholm) [reply docs/clients/ssm.md]: Thank you

**Comment by jderusse** [comment]
thank you

**Pull request merged**